src/index.html

```diff
@@ -7,8 +7,60 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link 
+      rel="preconnect" 
+      href="https://fonts.gstatic.com" 
+      crossorigin="anonymous"
+    >
+    <link 
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" 
+      rel="stylesheet"
+    >
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="style-header">
+      <a href="#Home.html">
+        <img 
+          class="logo nav__list" 
+          src="images/logo.png"
+          alt="MoyoLogo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item active-item">
+            <a class="nav__link is-active" href="#apple">Apple</a> 
+          </li>
+          <li class="nav__item">
```

Comment:
don`t forget to add empty line between sibling tags

Reply:
ok)